.packit.yaml

```diff
@@ -0,0 +1,52 @@
+specfile_path: packaging/leapp.spec
+
+upstream_package_name: leapp
+downstream_package_name: leapp
```

Comment:
These should be safe to remove, they both default to the name of the Github repository, i.e. `leapp`

```suggestion
```